RadixWallet/Clients/AuthorizedDappsClient/AuthorizedDappsClient+Interface.swift

```diff
@@ -118,56 +121,3 @@ extension AuthorizedDappsClient {
 		await (try? getAuthorizedDapps().contains { $0.id == address }) ?? false
 	}
 }
-
-extension Profile.Network.AuthorizedDapp.AuthorizedPersonaSimple.SharedPersonaData {
```

Comment:
moved to `Stage2MigrateToSargon+SharedPersonaData`: https://github.com/radixdlt/babylon-wallet-ios/blob/sargon_profile_integration_stage_1_batch_1/RadixWallet/ProfileSargonMigration/Stage2/Atoms/Stage2MigrateToSargon%2BSharedPersonaData.swift#L5